unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts

```diff
@@ -36,14 +36,20 @@ export class DgraphDataSource extends RESTDataSource {
   }
 
   private static getEntityCategoryString(type: string) {
-    switch(type) {
-      case EEntityType.DRUG:
-        return 'Drug';
-      case EEntityType.MEDICINAL_PRODUCT:
-        return 'Consumable';
-      case EEntityType.OTHER:
-        return 'Other';
-    }
+    const types = type.replace(/[[\]]+/g, '').split(',');
+    const categories = types.map(type => {
+      switch(type) {
+        case EEntityType.DRUG:
+          return 'Drug';
+        case EEntityType.MEDICINAL_PRODUCT:
+          return 'Consumable';
+        case EEntityType.OTHER:
+          return 'Other';
+        default:
+          return type;
+      }
+    });
+    return JSON.stringify(categories);
```

Comment:
Idea is to return types as a stringified array, makes the query a bit tidier!

Reply:
Nice, works well 👍